source/ProcessManager.Example.Orchestrations/Processes/BRS_101/UpdateMeteringPointConnectionState/V1/Activities/EnqueueActorMessagesActivity_Brs_101_UpdateMeteringPointConnectionState.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Energinet.DataHub.ProcessManager.Abstractions.Core.ValueObjects;
+using Energinet.DataHub.ProcessManager.Components.EnqueueActorMessages;
+using Energinet.DataHub.ProcessManager.Core.Application.Orchestration;
+using Energinet.DataHub.ProcessManager.Core.Domain.OrchestrationInstance;
+using Energinet.DataHub.ProcessManager.Example.Orchestrations.Abstractions.Processes.BRS_101.UpdateMeteringPointConnectionState.V1.Model;
+using Energinet.DataHub.ProcessManager.Shared.Api.Mappers;
+using Microsoft.Azure.Functions.Worker;
+
+namespace Energinet.DataHub.ProcessManager.Example.Orchestrations.Processes.BRS_101.UpdateMeteringPointConnectionState.V1.Activities;
+
+/// <summary>
+/// Enqueue messages in EDI (and set step to running)
```

Comment:
```suggestion
/// Enqueue messages in EDI
```
"EnqueueActorMessagesStep" has the responsibility of setting the step to running and terminating it?